src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -539,16 +546,20 @@ InvokerFactory
 ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::PrepareInvokerFactory(
     const ConvolutionContext& params, std::size_t ws_sz) const
 {
-#if(MIOPEN_BACKEND_HIP && MIOPEN_USE_ROCBLAS)
-    int flags         = 0;
-    int reserved      = 0;
-    int* reserved_ptr = nullptr;
-    int unused        = 0;
-    int N, C, H, W, K, n_groups, out_H, out_W, R, S;
-
-    GetCompiledInParameters(
-        params, &C, &K, &R, &S, &N, &n_groups, &H, &W, &out_H, &out_W, &unused, &unused);
-    // clang-format off
+#if(MIOPEN_BACKEND_HIP && (MIOPEN_USE_ROCBLAS || MIOPEN_USE_MIOPENTENSILE))
+#if(!MIOPEN_USE_ROCBLAS)
```

Comment:
This is not needed, please revert.

Reply:
Do you mean remove both #if? The first condition was originally there for rocblas. Was the related workaround been removed?

Reply:
Revised, not reverted yet